### ISSUE TITLE
Implement server batching

### ISSUE THREAD

**Issue by StellaAthena** [EleutherAI/GPTNeo#84]

**Comment by sdtblck** [comment]
see comment in return server logits issue
